src/classes/Visitor.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class Visitor
+{
+    protected $_name;
+    protected $_surname;
+
+    public function __construct($name, $surname)
+    {
+        $this->_name = $name;
+        $this->_surname = $surname;
+    }
+    public function returnFullName () {
+        return 'My name is ' . $this->_name . ' ' . $this->_surname;
+    }
```

Comment:
Inconsistent code style, let's do the same way as you did in the constructor: 

```suggestion
    public function returnFullName () 
    {
        return 'My name is ' . $this->_name . ' ' . $this->_surname;
    }
```